cellbender/remove_background/run.py

```diff
@@ -771,7 +789,12 @@ def run_inference(dataset_obj: SingleCellRNACountsDataset,
                 sys.exit(0)
             else:
                 logger.info(f'No more attempts are specified by --num-training-tries. '
-                            f'Therefore the workflow will abort here.')
+                            f'Therefore the workflow will run once more without ELBO restrictions.')
```

Comment:
Do you want to re-run it?  Does this re-run "cache", i.e. use the checkpoint?  Or does it actually retrain the whole thing?

Reply:
Yes, after the changes to the elements that are included in the hash computation, this uses the most recent checkpoint.

Reply:
Awesome, sounds perfect